core/src/main/scala/dagr/core/execsystem/TaskExecutionInfo.scala

```diff
@@ -24,11 +24,33 @@
 package dagr.core.execsystem
 
 import java.nio.file.Path
-import java.time.Instant
+import java.time.{Duration, Instant}
 
+import dagr.commons.util.TimeUtil._
 import dagr.core.DagrDef._
 import dagr.core.tasksystem.Task
 
+object TaskExecutionInfo {
+
+  /** Gets the total execution time and total time since submission, in seconds, or None if the task has not started and ended. */
+  def getDurationSinceStart(taskInfo: TaskExecutionInfo): Option[(Long, Long)] = (taskInfo.submissionDate, taskInfo.startDate, taskInfo.endDate) match {
```

Comment:
Why not just define these methods on the `TaskExecutionInfo` class?  I don't understand the benefit of having methods in the companion object that just take in and manipulate an instance of the companion class.  (If you do move them to the class, probably drop the `get` prefix :wink: )